app/api/v1/models/question_models.py

```diff
@@ -17,13 +17,48 @@ def save(self, meetup_id, question):
         question_record.append(data)
         return question_record
 
-class VotingRecords():
-    def save(self, question_id, vote):
+class VotingRecord():
+    def __init__(self):
+        self.rec = voting_record
+
+    def locate(self, id):
+        res = None
+        for item in self.rec:
+            if item['question_id'] == id:
+                res = item
+        return res
+
+    def save(self, id, vote):
+
         data = {
-        "vote_id": len(voting_record) + 1,
-        "question_id": question_id,
-        "voted-on": datetime.datetime.now(),
-        "vote": vote
+            "question_id" : id,
+            "upvotes" : 0,
+            "downvotes" : 0,
+            "voted_on" : datetime.datetime.now()
         }
-        voting_record.append(data)
-        return voting_record
+        #Executed when the vote parsed is True
+
+        if vote:
+            data['upvotes'] = data['upvotes'] + 1
+
+        #Executed when the vote parsed is True
```

Comment:
This comment seems to imply the opposite, did you mean false instead? Change it to false for it for when a question is downvoted.

Reply:
Thanks alot I have seen it .I will correct and place it  in the line below that executes the donvote